test/mpi/maint/test_coll_algos.sh

```diff
@@ -650,12 +661,16 @@ for algo_name in ${algo_names}; do
     #set the environment
     env="${testing_env} env=MPIR_CVAR_IALLTOALLW_INTRA_ALGORITHM=${algo_name}"
 
-    echo "alltoallw1 10 ${env}" >> ${testlist_cvar}
-    echo "alltoallw2 10 ${env}" >> ${testlist_cvar}
-    echo "alltoallw_zeros 1 ${env}" >> ${testlist_cvar}
-    echo "alltoallw_zeros 2 ${env}" >> ${testlist_cvar}
-    echo "alltoallw_zeros 5 ${env}" >> ${testlist_cvar}
-    echo "alltoallw_zeros 8 ${env}" >> ${testlist_cvar}
+    if ! [[ $algo_name =~ ^gentran_(inplace) ]] ; then
+        echo "alltoallw1 10 ${env}" >> ${testlist_cvar}
+    fi
+    if ! [[ $algo_name =~ ^gentran_(blocked|inplace) ]] ; then
+        echo "alltoallw2 10 ${env}" >> ${testlist_cvar}
```

Comment:
We can keep this for both, only half the test uses `in_place`, we should test the other half.